requre/online_replacing.py

```diff
@@ -15,132 +15,153 @@
     TEST_METHOD_REGEXP,
 )
 from requre.helpers.requests_response import RequestResponseHandling
-from requre.utils import get_datafile_filename
+from requre.utils import get_datafile_filename, get_module_of_previous_context
 from requre.helpers.guess_object import Guess
 
 logger = logging.getLogger(__name__)
 
 
+class ModuleRecord:
+    def __init__(self, what, parent, original, replacement):
+        self.what = what
+        self.parent = parent
+        self.original = original
+        self.replacement = replacement
+        self.caller_module = get_module_of_previous_context()
+
+
+def _match(
```

Comment:
Can we have some better name and/or some docsring?

I am not sure if  `_` should be used here...